docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -6262,7 +6262,7 @@ SELECT reinterpret(toInt8(-1), 'UInt8') as int_to_uint,
 
 Result:
 
-```
+```text
```

Comment:
Looking at the previous file in this PR, should we consistently use `response` for instead of `text` for quer results?

Reply:
We should we be consistent although from a syntax highlighting standpoint there is no difference.